pgbot/commands/base.py

```diff
@@ -540,6 +555,7 @@ async def call_cmd(self):
 
         # command has been blacklisted from running
         if cmd in db.DiscordDB("blacklist").get([]):
+            emotion.update_confused(time.time(), mode="cmd")
```

Comment:
Why have all of these separately? You could just put it in `handle_cmd`

Reply:
ah dang it, there's still a bunch of newlines in base.py that were left when I deleted the code